### DISCUSSION TITLE
nUMI column in metadata

### DISCUSSION THREAD

**Discussion by sjwu571** [satijalab/seurat#5798] [Q&A]
Hi everyone, 

I'm new to Seurat and working on exploring how to filter my gene expression dataset. I wish to filter my dataset based on UMI counts per cell. On a previous paper and one of your earlier tutorial (https://satijalab.org/seurat/archive/v2.4/pbmc3k_tutorial.html), there should be a "nUMI" column in the metadata. But in my Seurat object I didn't see that.

Also I found in Seurat there is a sampleUMI() function, which I ran as:

downsampled <- SampleUMI(pbmc[['RNA']]@counts, max.umi = 10000, upsample = FALSE, verbose = FALSE)

How do I know if this filtering step actually did anything? 

Thanks in advance!

JS

**Comment by Zig-E-Stardust** [comment]
I'm not expert, so I could be wrong, but I'm 80%-90% sure that what was described as "nUMI" in the 2018 guide you linked to is now (in the newer versions) called "nCount_RNA" (and what was described back then as nGene should now be "nFeature_RNA").
To look at the "nCount_RNA" and "nFeature_RNA", you can use, for instance, a violin plot:
VlnPlot(pbmc, features = c("nFeature_RNA", "nCount_RNA"), ncol = 2)
